dadi/lib/storage/disk.js

```diff
@@ -87,7 +89,7 @@ DiskStorage.prototype.put = function (stream, folderPath) {
 
         var data = {
```

Comment:
@eduardoboucas everything else has been `let`ed, shall we do it here, too?

Reply:
Absolutely! I'll do another round of `let`ting on these files.